docs/learn-faq.md

```diff
@@ -6,80 +6,327 @@ sidebar_label: Frequently Asked Questions (FAQs)
+of the parachain and the Polkadot network is completely separate from parachain
+economics. Parachains need collators to continue to progress, so it wouldn't be
+unreasonable to see them incentivize collator nodes in some way but it is completely
+up to parachain implementers.
+
+### Are parachains ephemeral? What happens when a parachain loses the next auction?
+
+Parachains are not ephemeral. As long as someone is keeping the data for a
+parachain, the parachain can move between being a parachain, a parathread, or
+a separate sovereign chain at different points of its lifetime. Especially
+with parathreads, parachains can be decommissioned to only produce blocks when
+their usage and throughput makes it necessary.
+
+When a parachain loses an auction for renewal, that parachain has a few options.
+In most cases, becoming a parathread instead would be a suitable choice. Parathreads
+are still secured by the Relay Chain, but don't need to hold a parachain slot
+and can produce a block when its economically feasible for them. For more on
+parachains please see the [parachains page](learn-parachains) and for more on
+parathreads see [here](learn-parathreads).
+
+## Networking
+
+### What is libp2p?
+
+[Libp2p][libp2p] is a modular and extensible networking stack that is used by
+IPFS, Substrate, and many other projects. It is a collection of peer-to-peer
+protocols for finding peers and connecting to them. It's modules have logic for
+content routing, peer routing, peer discovery, different transports, and NAT
+traversals. It is intended to be used by applications for building large scale
+peer-to-peer networks by only selecting the parts of the protocol suite that are
+needed.
+
+The Rust implementation of the specification was built and primarily maintained
+by a team of contributors at Parity Technologies. The Go and JavaScript versions
+are maintained by Protocol Labs as well as community contributors. It is an open
+source project that is actively developed and expanded on various code repositories
+hosted on [GitHub][libp2p github].
+
+### Does Polkadot use libp2p?
+
+Yes, since Polkadot is built with Substrate. Substrate uses a networking protocol
+that is based on libp2p (specifically the Rust libp2p library). However, Substrate
+uses a mix of standard libp2p protocols and protocols that are homegrown and not
+official libp2p standards. Of the standards protocols, these which are shared
+with other implementations of libp2p such as IPFS, are connection-checking (ping),
+asking for information on a peer (identity), and Kademlia random walks (kad).
+
+Of the protocols that are custom to Substrate, there are the legacy Substrate
+stream, a request-response for getting information on blocks (sync), a light
+client protocol, a notification protocol for transactions, and block announcement.
+For detailed information on how Substrate uses libp2p and the standard and custom
+protocols, please see the [networking documentation][substrate network]. 
```

Comment:
Is that "substrate network" link valid?

Reply:
Ahh just realized these are footnotes.  I think it might make sense to link directly instead of indirectly through footnotes but I could be convinced otherwise.

Reply:
They look the same when presented in HTML (inline links), just some markdown syntactic sugar.